scripts/scil_flip_streamlines.py

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Flip tracts around specific axes.
+
+IMPORTANT: this script should only be used in case of absolute necessity. It's
+better to fix the real tools than to force flipping tracts to have them fit in
+the tools.
+"""
+
+import argparse
+
+from dipy.io.stateful_tractogram import StatefulTractogram
+from dipy.io.streamline import save_tractogram
+import numpy as np
+
+from scilpy.io.streamlines import load_tractogram_with_reference
+from scilpy.io.utils import (add_reference_arg,
+                             add_overwrite_arg,
+                             assert_inputs_exist,
+                             assert_outputs_exist)
+
+
+def _build_args_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+
+    p.add_argument('in_tractogram',
+                   help='Path of the input tractogram file.')
+    p.add_argument('out_tractogram',
+                   help='Path of the output tractogram file.')
+
+    p.add_argument('axes', metavar='dimension',
+                   choices=['x', 'y', 'z'], nargs='+',
+                   help='The axes you want to flip. eg: to flip the x '
+                        'and y axes use: x y.')
+
+    add_reference_arg(p)
+    add_overwrite_arg(p)
+    return p
+
+
+def get_axis_flip_vector(flip_axes):
+    flip_vector = np.ones(3)
+    if 'x' in flip_axes:
+        flip_vector[0] = -1.0
+    if 'y' in flip_axes:
+        flip_vector[1] = -1.0
+    if 'z' in flip_axes:
+        flip_vector[2] = -1.0
+
+    return flip_vector
+
+
+def get_tracts_bounding_box(tracts):
```

Comment:
This could be moved to a module, it will probably be useful to other scripts.

Reply:
This function has been deleted and it is now included in StatefulTractogram.